File: After.old/Scripts/Models/Power.ts
namespace After.Models {
    export class Power {
        constructor() {
            this.TargetList = new Array<Targets>();
        }
        Name: string;
        Description: string;
        Category: string;
        MinRange: number;
        MaxRange: number;
        CanCrossZCoord: boolean;
        TargetList: Array<Targets>;
    }
    enum Targets {
        Self,
        Character,
        Player,
        Location
    }
}

File: After_Old/Code/Models/Power.cs
using System.Collections.Generic;

namespace After.Data
{
    public class Power
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public double MinRange { get; set; }
        public double MaxRange { get; set; }
        public bool CanCrossZCoord { get; set; }
        public List<Targets> TargetList { get; set; } = new List<Targets>();
        public enum Targets
        {
            Self,
            Character,
            Player,
            Location
        }
    }
}